src/main/java/com/puppycrawl/tools/checkstyle/utils/XpathUtil.java

```diff
@@ -87,6 +100,9 @@
             TokenTypes.IDENT, TokenTypes.STRING_LITERAL, TokenTypes.CHAR_LITERAL,
             TokenTypes.NUM_LONG, TokenTypes.NUM_INT, TokenTypes.NUM_DOUBLE, TokenTypes.NUM_FLOAT);
 
+    /** Delimiter to separate xpath results. */
+    private static final String DELIMITER = "---------" + System.lineSeparator();
```

Comment:
JDK name for this is delimiter, so I decided to stick to this terminology. Also, this delimiter includes line separator in it, it will look weird.
![image](https://user-images.githubusercontent.com/8901354/63947179-20b68280-ca7f-11e9-825f-ee76fb5fd383.png)